bokeh/embed/standalone.py

```diff
@@ -177,8 +176,13 @@ def components(models: Model | Sequence[Model] | Dict[str, Model], wrap_script:
         <script src="https://cdn.bokeh.org/bokeh/release/bokeh-gl-x.y.z.min.js"></script>
         <script src="https://cdn.bokeh.org/bokeh/release/bokeh-mathjax-x.y.z.min.js"></script>
 
-    Note that in Jupyter Notebooks, it is not possible to use components and show in
-    the same notebook cell.
+    The ``"-widgets"``, ``"-tables"``, and ``"-mathjax"`` files are only necessary
+    if your document includes :ref:`Bokeh widgets <userguide_interaction_widgets>`,
+    :ref:`data tables <userguide_interaction_widgets_examples_datatable>`, or
+    :ref:`math text <userguide_styling_math>`, respectively.
+
+    Note that in Jupyter notebooks, it is not possible to use ``components`` and
+    ``show`` in the same notebook cell.
```

Comment:
I wonder if this is actually still true. 🤷 Have not thought about it in years.

Reply:
Well, I just ran 

```
from bokeh.embed import components
from bokeh.io import output_notebook
from bokeh.plotting import figure, show
output_notebook() 

x = [1, 2, 3, 4, 5]
y = [6, 7, 2, 4, 5]

p = figure()

p.line(x, y, line_width=2)

show(p)
components(p)
```
In a standard Jupyter notebook, and it worked without any complaints - I got both the graph and the HTML code. Same thing in JupyterLab. I'll go ahead and remove this sentence then!